doc/classes/EditorScript.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Scripts extending this class and implementing its [method _run] method can be executed from the Script Editor's [b]File &gt; Run[/b] menu option (or by pressing [kbd]Ctrl + Shift + X[/kbd]) while the editor is running. This is useful for adding custom in-editor functionality to Godot. For more complex additions, consider using [EditorPlugin]s instead.
+		If a script extending this class also has a class name, it will show up in the editors command palette. The display name in the command palette can be customized by overriding [method _get_name].
```

Comment:
```suggestion
		If a script extending this class also has a global class name, it will be included in the editor's command palette. The display name in the command palette can be customized by overriding [method _get_name].
```

Reply:
From what I can see we use either "global class name" or "script class name" (+ one instance of "named class"). This is probably something we need to define once and for all and make consistent throughout class reference and docs. But just "class name" is ambiguous as all classes have named (e.g. here EditorScript is the name of the class being extended) - to a point that we often disambiguate with "[b]built-in[/b] class name" when talking about that one.

Reply:
CC @Mickeon

Reply:
Did go with you suggestions, so "global class name". Will leave the point unresolved in case there is some other naming standard already in use.

Reply:
Omg a direct mention.
It should be "_global class_" across the board, probably. Most people casually say it that way, too. However, EditorFileSystem & co. seem to call it "_script class_" first, which feels like legacy baggage prone to confusion.